packages/visx-demo/happo/gallery.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { tiles as examples } from '../src/components/Gallery';
+import AxisTile from '../src/components/Gallery/AxisTile';
+import XYChartTile from '../src/components/Gallery/XYChartTile';
+
+type HappoSnapshot = {
+  component: string;
+  variants: {
+    [key: string]: (
+      renderInDom: (component: React.ReactElement) => void,
+    ) => React.ReactNode | Promise<unknown>;
+  };
+};
+
+const MAX_TIMEOUT_MS = 400;
+const specialCases = new Set(['@visx/demo-axis', '@visx/demo-xychart']);
+
+function getComponentName(T: typeof examples[0]) {
+  return T.packageJson.name || 'missing-name';
+}
+
+const snapshots: HappoSnapshot[] = examples
+  .filter(Example => !specialCases.has(getComponentName(Example)))
+  .map(Example => ({
+    // note: this (reasonably) asserts Examples have unique names
+    component: getComponentName(Example),
+    variants: { default: () => <Example.default /> },
+  }));
+
+export default snapshots.concat([
+  // needs timeout for animated axes
+  {
+    component: '@visx/demo-axis',
+    variants: {
+      default: renderInDom => {
+        return new Promise(resolve => {
+          renderInDom(<AxisTile />);
+          setTimeout(() => resolve(), MAX_TIMEOUT_MS);
+        });
+      },
+    },
+  },
+  // needs timeout for animated axes
```

Comment:
One option that you might want to consider is to not use an animation if prefers-reduced-motion is set to reduce, and then add `prefersReducedMotion: true,` to your RemoteBrowserTarget:

```ts
'chrome-desktop': new RemoteBrowserTarget('chrome', {
  viewport: '800x552',
  prefersReducedMotion: true,
}),
```

Reply:
this is a cool idea. it'll be a bit of work to re-write those examples because the animations aren't css-based. looks like you can do the media query in JS to detect it, so I can swap out animated components for their static analogs. I may do this in a follow up PR depending on how big the change is.